src/RiakClient/Extensions/JsonExtensions.cs

```diff
@@ -21,18 +21,42 @@ namespace RiakClient.Extensions
 {
     using Newtonsoft.Json;
 
+    /// <summary>
+    /// Helper extension methods to use common Newtonsoft.Json methods with less code. 
+    /// </summary>
     public static class JsonExtensions
```

Comment:
With internal, no docs needed.  Not sure if we have any customers using it though.